modules/tensor_mechanics/src/materials/LAROMANCEStressUpdateBase.C

```diff
@@ -143,6 +153,10 @@ LAROMANCEStressUpdateBaseTempl<is_ad>::validParams()
   params.addParam<unsigned int>("seed", 0, "Random number generator seed");
   params.addParam<std::string>("stress_unit", "Pa", "unit of stress");
 
+  // use std::string here to avoid automatic absolute path expansion
+  params.addParam<FileName>("model", "LaRomance model JSON datafile");
```

Comment:
Does this input parameter need to be suppressed in the downstream .C files? I kinda think we should just move to json completely asap.

Reply:
I'm planning to move to JSON as soon as this is merged and will keep the old objects around for a while, with the `model` parameter set to their respective data file and _then_ suppressed.